transmit-lib/src/cmr/transmit/metadata_db.clj

```diff
@@ -331,7 +331,15 @@
                                 :throw-exceptions false
                                 :headers (ch/context->http-headers context)}))
         status (int (:status response))
-        body (json/decode (:body response))
+        ;; For CMR-4841 - log the first 255 characters of the response body if
+        ;; the parsing of the html throws exception. 
+        response-body (:body response)
```

Comment:
These should be moved to the catch block.

Reply:
Why?

Reply:
Didn't pay attention. So move the comment and response-body-logged to the catch block.